### PR TITLE
Feature/mo pub bidder support banner

### PR DESCRIPTION
**Pull request by orkhanalizada2020**
AdBidding will be added separately on another branch

### CODE REVIEW

File: PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m [hunk 1]
```diff
@@ -22,19 +22,23 @@
...
+@interface AMAppMonetAdView () <HyBidAdViewDelegate, HyBidAdRequestDelegate>
 @property (nonatomic) CGSize size;
 @end
 
 @implementation AMAppMonetAdView
 
-- (id)initWithAdUnitId:(NSString *)adUnitId size:(CGSize)size {
-    HyBidAdSize *newSize = [[[HyBidAdSize alloc] init] convertSizeToHyBidAdSize:size];
-    self = [self initWithSize:newSize];
+- (id)initWithAdUnitId:(NSString *)adUnitId size:(HyBidAdSize *)size {
```
Comment (eros902002) [inline PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m:38]: Here we shouldn't replace the interface with our class. We should do the mapping internally.

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m:38]: Changing this from CGSize to HyBidAdSize will break existing publisher's integration

Comment (orkhanalizada2020) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m:38]: @eros902002 true! I changed it to test something and forgot return back. One moment :)

Comment (orkhanalizada2020) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m:38]: @can-soykarafakili yes! I forgot to change it back 🌚

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m:38]: @orkhanalizada2020 I think you can copy & paste the size mapping code that I've wrote in `AMCustomEventBanner` under AppMonet Adapters/MoPub Adapters 👍

Comment (orkhanalizada2020) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m]: @can-soykarafakili I've added it in my latest commit. Please check it

Comment (can-soykarafakili) [reply PubnativeLite/PubnativeLite/AppMonet/AMAppMonetAdView.m]: Looks okay @orkhanalizada2020  👍

**Review by can-soykarafakili — APPROVED** [review]
Looks good! For bidding, you can use `AppMonet+MoPub` extension class to add your methods👍 If you want to see code completion, just don't forget to add `AppMonet+MoPub` extension class to HyBid Demo App target and remove it back when you are done 😉

**Review by eros902002 — APPROVED** [review]
Looks good

**Pull request merged**